src/components/404/ComingSoon.tsx

```diff
@@ -4,59 +4,24 @@ import Link from "next/link";
 import { useRef } from "react";
 import { useInView } from "framer-motion";
 import { Button } from "@/components/ui/button";
+import { Bars } from "@/components/ui/bars";
 
 export const ComingSoon = () => {
-  const barHeight = 75;
   const headerRef = useRef(null);
   const isInView = useInView(headerRef, { once: false });
 
-  const leftBars = Array.from({ length: 3 }, (_, index) => {
-    const top = index * (barHeight * 2) + barHeight;
-
-    return (
-      <div
-        key={`left-${index}`}
-        className="absolute w-[60%] bg-black z-[-1]"
-        style={{
-          top: `${top}px`,
-          left: `${isInView ? 0 : "-100%"}`,
-          height: `${barHeight}px`,
-          transition: "left 1.5s",
-        }}
-      ></div>
-    );
-  });
-
-  const rightBars = Array.from({ length: 4 }, (_, index) => {
-    const top = index * (barHeight * 2);
-
-    return (
-      <div
-        key={`right-${index}`}
-        className="absolute w-[35%] bg-black z-[-1]"
-        style={{
-          top: `${top}px`,
-          right: `${isInView ? 0 : "-100%"}`,
-          height: `${barHeight}px`,
-          transition: "right 1.5s",
-        }}
-      ></div>
-    );
-  });
-
```

Comment:
@woynowskiy instead of duplication these bars logic i created <Bars/> component, please get familiar with it